src/qt/bitcoinunits.cpp

```diff
@@ -40,9 +40,9 @@ QString BitcoinUnits::name(int unit)
 {
     switch(unit)
     {
-    case BTC: return QString("BTC");
-    case mBTC: return QString("mBTC");
-    case uBTC: return QString::fromUtf8("μBTC");
+    case BTC: return QString(CURRENCY_UNIT.c_str());
+    case mBTC: return QString(("m" + CURRENCY_UNIT).c_str());
```

Comment:
No longer relevant here, but for next time: you can use QString::fromStdString to convert a std::string to QString. No need for c_str.